### PR TITLE
Add warning about installing traits and traitsui

### PR DESCRIPTION
**Pull request by francisco-dlp**
As of today Anaconda does not distribute `traits` and `traitsui` for Python 3. Therefore, installing HyperSpy in Anaconda is harder than it should be for most Windows users, maybe MacOS users and those Linux users without dev tools installed. This adds a warning about this issue in the User Guide.

See [this post in the mailing list](https://groups.google.com/d/msg/hyperspy-users/lUna5X3vMLQ/hOxVBMHWCwAJ) reporting the issue.

### CODE REVIEW

File: doc/user_guide/install.rst [hunk 1]
```diff
@@ -51,6 +51,31 @@ Quick instructions to install HyperSpy using Anaconda (Linux, MacOs, Windows)
        $ pip install hyperspy
 
 .. warning::
+
+    HyperSpy requires two packages, `traits` and `traitsui` which are not
+    written in pure Python and, therefore, needs to be compiled. The above
```
Comment (dnjohnstone) [inline doc/user_guide/install.rst]: grammar -- "need to be compiled"

File: doc/user_guide/install.rst [hunk 2]
```diff
@@ -51,6 +51,31 @@ Quick instructions to install HyperSpy using Anaconda (Linux, MacOs, Windows)
...
+    command will download these packages and try to compile them automatically.
+    However, if the required development tools are not installed in your system
+    the installation will fail. This is more likely to be the case in Microsoft
+    Windows and (to a lesser extend) MacOS. Microsoft Windows users could
```
Comment (dnjohnstone) [inline doc/user_guide/install.rst]: "to a lesser extent"

File: doc/user_guide/install.rst [hunk 3]
```diff
@@ -51,6 +51,31 @@ Quick instructions to install HyperSpy using Anaconda (Linux, MacOs, Windows)
...
+    install the HyperSpy Bundle instead which includes all the required
+    libraries. Alternatively, they could download and install `traits` and
+    `traitsui` from `Christoph Gohlke's repository
+    <http://www.lfd.uci.edu/~gohlke/pythonlibs/>`_.
+
+    As of the 27th of May 2016, Anaconda does not distribute `traits` and
+    `traitsui` for Python 3. Once it does, the simples solution is to install
+    those packages using `conda` before installing HyperSpy as above e.g.:
```
Comment (dnjohnstone) [inline doc/user_guide/install.rst]: The section above is a little unclear... You mean that once Anaconda stops distributing the packages you should install using conda?

**Comment by dnjohnstone** [comment]
Same comments apply to the copied section obviously.

**Comment by to266** [comment]
[conda-forge](https://conda-forge.github.io/feedstocks.html
) does, though: [traits](https://github.com/conda-forge/traits-feedstock), [traitsui](https://github.com/conda-forge/traitsui-feedstock)

So installing both of these packages is as simple as 
`conda install -c conda-forge traits traitsui`

(and I've just tested it on a virtual windows machine without all the compiling libraries - it seemed to work perfectly)

FYI I'm trying to add hyperspy to the conda-forge as well, however in doing so I seem to have encountered a bug in their build system. Will let you know when it's done

**Comment by francisco-dlp** [comment]
Great, I've updated the docs to recommend installing traits and traitsui using conda-forge instead.

**Comment by vidartf** [comment]
@to266 Do you want to reply to the mailing list with your solution?

**Comment by to266** [comment]
I just did, thanks for the reminder @vidartf !

**Pull request merged**